src/test/rustdoc-ui/invalid-syntax.stderr

```diff
@@ -7,6 +7,7 @@ LL | | /// \__________pkt->size___________/          \_result->size_/ \__pkt->si
 LL | | /// ```
    | |_______^
    |
+   = note: `#[warn(invalid_rust_codeblock)]` on by default
```

Comment:
Could you add some tests for when `ignore` is present (and for all the rest of the branching you have for the diagnostics)?